### ISSUE TITLE
Dplay.se - Now need login to download

### ISSUE THREAD

**Issue by Sopor** [spaam/svtplay-dl#1234]
It seems that Dplay.se now force you to login to download videos and you need to logout from your web browser or else it won't work.

Anyone that can confirm this?

**Comment by ecsjjgg** [comment]
Yes I can confirm this...

**Comment by Pirrevirr** [comment]
Yes I can also confirm this, but I can't get it to work at all by logging out from my web browser. I've tried everything from:

Staying logged on and svtplay-dl [url]
Staying logged on and svtplay-dl -u login -p password [url]
Logging out from web browser and svtplay-dl [url]
Logging out from web browser and svtplay-dl -u login -p password [url]

Every option gives me: "ERROR: No videos found. You dont have permission to watch this".

Which of these options are you suggesting and what am I then doing wrong?

Thank you in advance

**Comment by Bengt-H** [comment]
I believe they also have added a captcha (a puzzle you have to solve when logging in) which gives this warning and error:
**WARNING: Wrong username/password. no support for recaptcha.**
**ERROR: No videos found. You dont have permission to watch this**

**Comment by thomassandstrom** [comment]
I also got "ERROR: No videos found. You dont have permission to watch this" when trying to download from dplay.se
No matter if I was logged in or logged out.

**Comment by kjetilpp** [comment]
Any updates :) ?

**Comment by thomassandstrom** [comment]
> Any updates :) ?

I switched to youtube-dl instead.
It works if you copy your cookie-file.

**Comment by kjetilpp** [comment]
Nice! 

How to get the episode url? 

> > Any updates :) ?
> 
> I switched to youtube-dl instead.
> It works if you copy your cookie-file.

**Comment by thomassandstrom** [comment]
> Nice!
> 
> How to get the episode url?
> 
> > > Any updates :) ?
> > 
> > 
> > I switched to youtube-dl instead.
> > It works if you copy your cookie-file.

For getting the episode url's, I use svtplay-dl with the options for getting all episodes,  only get the episodes url, username and password.

To get the show list for "Aldrig vuxen" at the adress https://www.dplay.se/program/aldrig-vuxen, I use the following line:
`svtplay-dl -u USERNAME -p PASSWORD -A --get-only-episode-url https://www.dplay.se/program/aldrig-vuxen`

It results in a list like this :
`INFO: Episode 1 of 12
INFO: Url: https://www.dplay.se/videos/aldrig-vuxen/avsnitt-1
INFO: Episode 2 of 12
INFO: Url: https://www.dplay.se/videos/aldrig-vuxen/avsnitt-2
INFO: Episode 3 of 12
INFO: Url: https://www.dplay.se/videos/aldrig-vuxen/avsnitt-3
INFO: Episode 4 of 12
INFO: Url: https://www.dplay.se/videos/aldrig-vuxen/avsnitt-4
INFO: Episode 5 of 12
INFO: Url: https://www.dplay.se/videos/aldrig-vuxen/avsnitt-5
INFO: Episode 6 of 12
INFO: Url: https://www.dplay.se/videos/aldrig-vuxen/avsnitt-6
INFO: Episode 7 of 12
INFO: Url: https://www.dplay.se/videos/aldrig-vuxen/avsnitt-7
INFO: Episode 8 of 12
INFO: Url: https://www.dplay.se/videos/aldrig-vuxen/avsnitt-8
INFO: Episode 9 of 12
INFO: Url: https://www.dplay.se/videos/aldrig-vuxen/avsnitt-9
INFO: Episode 10 of 12
INFO: Url: https://www.dplay.se/videos/aldrig-vuxen/avsnitt-10
INFO: Episode 11 of 12
INFO: Url: https://www.dplay.se/videos/aldrig-vuxen/avsnitt-11
INFO: Episode 12 of 12
INFO: Url: https://www.dplay.se/videos/aldrig-vuxen/avsnitt-12`

This list is easy to copy to Notepad and edit so you just have the url's.
I then use this list for batch downloading episodes during the night.
